leapp/actors/__init__.py

```diff
@@ -86,6 +92,7 @@ def serialize(self):
             'path': os.path.dirname(sys.modules[type(self).__module__].__file__),
             'class_name': type(self).__name__,
             'description': self.description or type(self).__doc__,
+            'config_schemas': [c.__name__ for c in self.config_schemas],
```

Comment:
I am thinking here, whether we only add the name of the config or a whole serialized representation as we do with dialogs. In my head ``c.serialize()`` makes more sense, however it is true that the schema is an entity that can be shared across actors opposed to dialog which is always specific to the given actor, so doing it this way has merit also. What do you think? @abadger @pirat89

Reply:
If we keep it like this, this also brings to my attention the issue where we store config information in leappdb. Is config part of actor metadata? I argue that the config scheme (the name) is actor metadata, however the schema itself is not. Maybe we should create another table and store the schema information there instead of in metadata.

Reply:
I'm not sure whether name only or a serialization of the schema class is appropriate here.  @pirat89 ?

I'm not sure what leappdb is used for either:  Would we want to store both the config_schema in it and the actual actor config settings?  Would we want to store just the config settings from config files or would we want to store the config dictionary that has been expanded to include default values as well?

Reply:
The reason we would like to have this information in the database is for debugging and troubleshooting purposes (using for example leapp-inspector). My idea is that we would like to have a snapshot of the leapp configuration just as the user sees it. This means that in case something goes wrong for the user we can see what is the state of leapp when he provides the leapp.db file.

So I personally would store both the entire schema, so we can see the setup of the configuration as well as the actual values provided by the user. The format in which this is stored does not matters from my point of view only in terms of how easily we can work with it later when this information is retrieved from processing by other tools (such as the leapp-inspector I mentioned)